### PR TITLE
Add CPE for WRLinux

### PR DESCRIPTION
**Pull request by radzy**
Signed-off-by: T.O. Radzy Radzykewycz <radzy@windriver.com>

### CODE REVIEW

File: WRLinux/input/oval/platform/wrlinux-cpe-dictionary.xml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<cpe-list xmlns="http://cpe.mitre.org/dictionary/2.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://cpe.mitre.org/dictionary/2.0 http://cpe.mitre.org/files/cpe-dictionary_2.1.xsd">
+      <cpe-item name="cpe:/o:windriver:wrlinux">
```
Comment (redhatrises) [inline WRLinux/input/oval/platform/wrlinux-cpe-dictionary.xml]: This CPE is not a valid NIST CPE database. Please remove.

File: WRLinux/input/oval/platform/wrlinux-cpe-dictionary.xml [hunk 2]
```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<cpe-list xmlns="http://cpe.mitre.org/dictionary/2.0"
+          xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+          xsi:schemaLocation="http://cpe.mitre.org/dictionary/2.0 http://cpe.mitre.org/files/cpe-dictionary_2.1.xsd">
+      <cpe-item name="cpe:/o:windriver:wrlinux">        
+            <title xml:lang="en-us">Wind River Linux all versions</title>        
+            <check system="http://oval.mitre.org/XMLSchema/oval-definitions-5" href="filename">installed_OS_is_wrlinux</check>
```
Comment (redhatrises) [inline WRLinux/input/oval/platform/wrlinux-cpe-dictionary.xml]: This CPE is not a valid NIST CPE database. Please remove.

**Comment by mpreisler** [comment]
Just for reference, the NIST CPE dict has 2 WRLinux related CPE IDs:

```
$ wget https://static.nvd.nist.gov/feeds/xml/cpe/dictionary/official-cpe-dictionary_v2.3.xml
$ grep wrlinux official-cpe-dictionary_v2.3.xml 
  <cpe-item name="cpe:/o:windriver:wrlinux:8.0">
    <cpe-23:cpe23-item name="cpe:2.3:o:windriver:wrlinux:8.0:*:*:*:*:*:*:*"/>
  <cpe-item name="cpe:/o:windriver:wrlinux:9.0">
    <cpe-23:cpe23-item name="cpe:2.3:o:windriver:wrlinux:9.0:*:*:*:*:*:*:*"/>
```

**Comment by radzy** [comment]
Thanks @mpreisler @redhatrises .  I didn't realize they had to be the same.

**Comment by mpreisler** [comment]
Looks good now, ACK.

**Pull request merged**